src/movableCoord.js

```diff
@@ -229,6 +219,23 @@ eg.module("movableCoord", [eg, window, "Hammer"], function(ns, global, HM) {
 			return this;
 		},
 
+		/**
+		 * get a hammer instance from elements using the eg.MovableCoord module.
+		 * @ko eg.MovableCoord 모듈을 사용하는 엘리먼트에서 hammer 객체를 얻는다
+		 * @method eg.MovableCoord#getHammer
+		 * @param {HTMLElement|String|jQuery} element An element from which the eg.MovableCoord module is using<ko>eg.MovableCoord 모듈을 사용하는 엘리먼트</ko>
+		 * @return {Hammer|null} An instance of Hammer.JS<ko>Hammer.JS의 인스턴스</ko>
+		 */
+		getHammer: function(element) {
+			var el = this._getEl(element);
+			var key = el[MC._KEY];
+			if (key) {
+				return this._hammers[key].inst;
```

Comment:
There can be a case of `this._hammers[key]` is `undefined`.
So adding condition is safer like
```
if (key && this._hammers[key]) {
    return this._hammers[key].inst;
} else {
```